fastlane/lib/fastlane/actions/spm.rb

```diff
@@ -10,6 +10,7 @@ def self.run(params)
         cmd << "--configuration #{params[:configuration]}" if params[:configuration]
         cmd << "--disable-sandbox" if params[:disable_sandbox]
         cmd << "--verbose" if params[:verbose]
+        cmd << "--very-verbose" if params[:very_verbose]
```

Comment:
Can we please add a unit test for it?  
You can take an inspiration from an existing tests in `spm_spec` for `--vebose` flag 💪

Reply:
Sorry for late reply 🙏🏼 
I will add a unit test!